### ISSUE TITLE
Binary files in feedstocks

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/conda-forge.github.io#128]
I have started to see this creep up of late and would really like us to figure out a way to avoid it entirely.

**Comment by ericdill** [comment]
What is the use case for including binary files in feedstocks?

**Comment by jakirkham** [comment]
Thus far, of the cases that I have seen, icons and test data. Not sure what else lies out there as I have had a chance to do a good survey.

Test data isn't too bad normally as it isn't being packaged FWICT. Though I believe basemap requires packaged data, which is basically impossible to workaround.

Icons are potentially worrisome as they are shipped, but maybe I'm being too paranoid.

**Comment by msarahan** [comment]
The worrisome part is copyright, or security, or what?

**Comment by jakirkham** [comment]
In the cases of icons thus far (though I don't expect this to hold for ever), it is being done by developers on the project or maintainers in close contact with the developers. So, they at least are getting approval and conceivably they know how it was made. The concern is I don't expect this trend to hold.

**Comment by msarahan** [comment]
Fair enough.  Icons don't necessarily come from the same place as the source, and they can and often do have different licenses.  I think we need to ask that people at least state where things like this come from, and any license that may be associated.

**Comment by ChrisBarker-NOAA** [comment]
I'm confused as to why any of this would be in the feedstock, rather than
in the source of the package itself -- what am I missing??

-CHB


On Mon, May 9, 2016 at 7:08 PM, Mike Sarahan <notifications@github.com>
wrote:

> Fair enough. Icons don't necessarily come from the same place as the
> source, and they can and often do have different licenses. I think we need
> to ask that people at least state where things like this come from, and any
> license that may be associated.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/conda-forge.github.io/issues/128#issuecomment-218041228>
>



-- 

Christopher Barker, Ph.D.
Oceanographer

Emergency Response Division
NOAA/NOS/OR&R            (206) 526-6959   voice
7600 Sand Point Way NE   (206) 526-6329   fax
Seattle, WA  98115       (206) 526-6317   main reception

Chris.Barker@noaa.gov

**Comment by ericdill** [comment]
One possible use case that just came up is in the audio processing package, [yaafe](https://github.com/conda-forge/staged-recipes/pull/2709). `yaafe` bundled a .wav file to smoketest the yafee cli during the test phase of conda build. I am not voting for or against the inclusion of binary files by making this comment. Just trying to surface another use case that wasn't already present in this issue. xref: https://github.com/conda-forge/conda-smithy/issues/500

**Comment by kain88-de** [comment]
For MDAnalysis we have an extra tests package that contains a lot of free simulation data. That has proven really valuable for us. We use it to showcase new features in small examples and users take them to give minimal example for bugs.
